spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/config/RabbitBinderConfigurationProperties.java

```diff
@@ -31,15 +31,15 @@
 
 	private int backOffMaxInterval;
 
-	private int backOffMultiplier;
+	private double backOffMultiplier;
 
 	private boolean transacted;
 
 	private boolean concurrency;
 
-	private MessageDeliveryMode defaultDeliveryMode;
+	private MessageDeliveryMode deliveryMode;
```

Comment:
I am not fully tuned in for s-c-s, but in the XD case, they are bus-level defaults, which can be overridden at the stream level. In the case of `getDefaultDefaultDeliveryMode` it's strictly correct because there's a default at the bus level, default at the stream level (overrides bus) and delivery mode at the message level (overrides both).

If s-c-s only has one output channel then I suppose, we can dump the default.

However, I thought the intention is to support multiple output channels (and we also have taps etc), so they really are bus defaults that might be overridden by the stream definition.